contracts/cw3-fixed-multisig/src/contract.rs

```diff
@@ -401,46 +401,56 @@ fn calc_range_start(start_after: Option<HumanAddr>) -> Option<Vec<u8>> {
 
 #[cfg(test)]
 mod tests {
+    use cosmwasm_std::from_binary;
     use cosmwasm_std::testing::{mock_dependencies, mock_env};
+
     use cw0::Duration;
     use cw2::{get_contract_version, ContractVersion};
 
     use crate::msg::Voter;
 
     use super::*;
-    use cosmwasm_std::from_binary;
+
+    fn voter<T: Into<HumanAddr>>(addr: T, weight: u64) -> Voter {
+        Voter {
+            addr: addr.into(),
+            weight,
+        }
+    }
 
     // this will set up the init for other tests
     fn setup_test_case<S: Storage, A: Api, Q: Querier>(
-        deps: &mut Extern<S, A, Q>,
+        mut deps: &mut Extern<S, A, Q>,
         env: Env,
-        voters: Vec<Voter>,
         required_weight: u64,
         max_voting_period: Duration,
     ) {
-        // Init a contract with admins
+        // Init a contract with voters
+        let voters = vec![
+            voter(env.message.sender.clone(), 1),
```

Comment:
Minor point, I think you can use `&env.message.sender` here which is *slightly* less verbose. Both are equivalent though